cubic-server/Client.cpp

```diff
@@ -320,3 +332,13 @@ void Client::sendChatMessageResponse(const protocol::PlayerChatMessage &packet)
 
     LDEBUG("Sent a chat message response");
 }
+
+void Client::sendDisconnectPlayerReason(const std::string &json)
+{
+    auto pck = protocol::DisconnectPlayerReason({
+        json
+    });
+    _sendData(*pck);
+
+    LDEBUG("Sent a disconnet reason for player");
```

Comment:
The wording here is a bit weird, why not something like
`Sent a disconnect player reason`
That would be coherent with the rest of the debug logs here.